src/System.Net.Sockets.Legacy/System.Net.Sockets.Legacy.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
@SidharthNabar can address this better than I can: IIUC it has to do with plans to add a package that provides some legacy Socket APIs that are excluded from the newer surface area (e.g. the APM APIs).